packages/front-end/components/Experiment/AnalysisSettingsBar.tsx

```diff
@@ -346,6 +355,10 @@ export default function AnalysisSettingsBar({
                       model={latest}
                       icon="refresh"
                       color="outline-primary"
+                      onSubmit={() => {
+                        setBaselineRow?.(0);
+                        setVariationFilter?.([]);
```

Comment:
Eventually we'll want to probably just have a general "reset analysis state" method or something? Maybe NBD since it's only in two places for now.

The more I think about this, the less certain I am about wanting to reset this automatically. Maybe the correct flow is this:
1) kick off queries
2) run standard default analysis
3) run the current state analysis

And then also have a "reset to default" setting somewhere? IDK if that's too painful, but it might be useful to ensure people within an org aren't confusing each other with screenshots with different filters applied.

Reply:
removed this in favor of persisting in the baselinechooser